src/components/Footer.jsx

```diff
@@ -150,10 +159,49 @@ function Footer() {
                 </svg>
                 <span className="sr-only">Dribbble account</span>
               </a>
+              {/* More social media icons ... */}
+              {/* FAQ button */}
+              <button
+                onClick={toggleModal}
+                className="text-gray-500 hover:text-gray-900 dark:hover:text-white ms-5"
+              >
+                FAQ
+              </button>
             </div>
           </div>
         </div>
       </footer>
+
+      {/* FAQ Modal */}
+      {isModalOpen && (
+        <div className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-50">
+          <div className="bg-white dark:bg-gray-800 p-5 rounded shadow-lg">
+            <h2 className="text-lg font-semibold text-gray-900 dark:text-white mb-4">
+              Frequently Asked Questions
+            </h2>
+            <div>
+              <h3 className="font-semibold">Q1: What is GlobeHoppin?</h3>
+              <p className="text-gray-600 dark:text-gray-400">
+                GlobeHoppin is a platform for discovering travel resources and connecting with fellow travelers.
+              </p>
+              <h3 className="font-semibold mt-3">Q2: How can I contact support?</h3>
+              <p className="text-gray-600 dark:text-gray-400">
+                You can contact support through our contact page.
+              </p>
+              <h3 className="font-semibold mt-3">Q3: Where can I find the terms and conditions?</h3>
+              <p className="text-gray-600 dark:text-gray-400">
+                The terms and conditions are available in the footer section of our website.
+              </p>
```

Comment:
These questions and answers should be combined into a single json/js file, exported from there, imported here, and looped through to display.

Reply:
okay